packages/oats-runtime/src/client.ts

```diff
@@ -184,7 +184,8 @@ function makeMethod(adapter: ClientAdapter, handler: server.Handler, pathParams:
     handler.body,
     handler.response,
     ctx =>
-      adapter({ ...ctx, path: fillInPathParams(params, handler.path), servers: handler.servers })
+      adapter({ ...ctx, path: fillInPathParams(params, handler.path), servers: handler.servers }),
```

Comment:
we tell oats that we are on client side so that safe knows what values to deserialize and what to serialize.